android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -254,7 +254,7 @@ public void setSrc(final String uriString, final String type, final boolean isNe
                     headers.putAll(toStringMap(mRequestHeaders));
                 }
 
-                setDataSource(uriString);
+                setDataSource(mThemedReactContext, parsedUrl, headers);
```

Comment:
In master (of the main repo) no headers are being passed to `setDataSource`. Even though the headers `HashMap` is being created (lines 247-251) and the cookies are added to it.

Is it me or was the headers never used? 

Luckily `setDataSource` had an overload that accepted headers which I could pass them to.